CMakeLists.txt

```diff
@@ -132,6 +132,28 @@ option(BUILD_DOCS "Build Api documentation")
 cvc5_option(STATIC_BINARY "Link against static system libraries \
 (enabled by default for static builds)")
 
+# Custom flags for WebAssembly compilation
+set(WASM_FLAGS "" CACHE STRING "Link flags for the WebAssembly binary generation")
```

Comment:
When do we need these flags and where are they set? Is this purely a user option?

Reply:
We need these flags when compiling into Web Assembly and need to pass some Emscripten flags to the compilation link. It's a user option so it is set in the configure.sh. The user needs to pass it as a string wrapped by single quote just like that: --wasm-flags='-s INCOMING_MODULE_JS_API=arguments -s INVOKE_RUN=1 -s MODULARIZE -s EXIT_RUNTIME=0 -s ENVIRONMENT=web'.

These flags modify how the wasm, glue code and HTML file are generated and behave.